front-end/src/components/GovernanceCard.tsx

```diff
@@ -38,9 +38,9 @@ const GovernanceAction = function ({
 				</Segment>
 				<Segment>
 					<h4 className={'proposalTitle'}>{title}</h4>
-					<CreationLabel
-						address={address}
-						text={'proposed'}
+					<Address
+						address={address!}
```

Comment:
We shouldn't have this `!`
https://stackoverflow.com/questions/47704652/what-does-exclamation-point-after-variable-mean-in-javascript

Why is that needed here?

Reply:
Ah, because `address` was optional in Props - let me change

Reply:
Actually, Typescript yells as the address prop in the Address component isn't optional, and it probably should stay that way.

Reply:
Hmm I don't get it, you did remove it in the end right? by making the address not optional.

Reply:
Yes, but that was without checking in browser :/ `address={address || ''}` would also be a workaround.

Reply:
ok, anything is better than `!`. This should only be used to tell the compiler that it's wrong and you know better. In this case the compiler was most probably right and the address can be undefined it seems. so the default `|| ''` is good.